docs/get-started/Join-the-Testnet.md

```diff
@@ -1,44 +1,25 @@
 # IRISnet Testnet Codename Fuxi
 
-## What is IRISnet
+## Fuxi  Testnet
 
-IRIS network (a.k.a. IRISnet) is named after Greek goddess **Iris**, said to be the personification of the rainbow and the faithful messenger between heaven and humanity. IRIS network aims to build the foundation which facilitates construction of distributed business applications. IRIS hub will provide iServices, which allow resources such as data service and computing services being invoked across blockchains. To know more about IRISnet, please read this blog.
+Fuxi testnet is used for testing irishub.
```

Comment:
The Fuxi testnet is used to provide a stable test environment for developers such as third-party wallets, browsers and other applications.